### DISCUSSION TITLE
Suggestion: GNU radio block support

### DISCUSSION THREAD

**Discussion by zouppen** [windytan/redsea#107] [Ideas]
Since there are C++ bindings for GNU Radio, I'm wondering if it would be straightforward to add support for GNU Radio (out tree module), having demodulated audio as sink and having tagged stream output instead of JSON content.

**Comment by zouppen** [comment]
Did a bit hacky GNU radio integration in my project https://github.com/zouppen/labiuutiset . It forks `redsea` and feeds data to it. Not suitable for integration to this project but just to show what can be done with such integration.

**Comment by windytan** [comment]
Interesting! I'll have to try and figure out how such a tagging module could be written.
